src/screens/Sidebar/common/index.js

```diff
@@ -1,6 +1,15 @@
 import SidebarDropdown from './SidebarDropdown';
 import SidebarHeader from './SidebarHeader';
-import SidebarListItem, { ItemSeparator } from './SidebarListItem';
+import SidebarListItem, { ItemSeparator, WidthFixer } from './SidebarListItem';
```

Comment:
I would prefer that everything is a named export, this is really crummy to read :(